keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -308,7 +314,7 @@ def compile(
         super().compile(loss=losses, **kwargs)
 
     def compute_loss(self, images, gt_boxes, gt_classes, training):
-        box_pred, cls_pred = self._forward(images, training=training)
+        box_pred, cls_pred = self.call(images, training=training)
```

Comment:
if you use `call` here, would the validation loss be out of order (compared to the train loss)?

Reply:
(as the validation_data would use `test_step` which calls `call` with training=False, so the decoded box is used to compute validation loss?

Reply:
Good catch -- I think this should be calling `super().call` (which should be identical to `_forward`)

I updated it to call super().call and the validation loss looks correct.
Without the change, the validation loss is super high (as one might expect)